app/models/pipeline_run.rb

```diff
@@ -555,13 +555,17 @@ def db_load_amr_counts
           allele: row["allele"],
           coverage: row["coverage"],
           depth: row["depth"],
-          annotation_gene: row["annotation"].split(";")[2],
-          genbank_accession: row["annotation"].split(";")[4],
           drug_family: row["gene_family"],
           total_reads: row["total_reads"],
           rpm: row["rpm"],
           dpm: row["dpm"],
         }
+        if row["annotation"]
```

Comment:
It looks like there's another bug here. You're doing `amr_results_table.each row` and then adding an entry to amr_counts_array for each row. But then `if annotation` you would add another row to amr_counts_array.

Reply:
You're right. I'm tired lol. I'm glad I have you all to do code review.